app/views/partials/_post.html.erb

```diff
@@ -0,0 +1,14 @@
+<li>
+ <h3> <% post.title%> </h3>
+ <p> <%= truncate(post.text, :length => 150) %> </p>
+ <strong>
+  <p class="placeholder stats">
+   Comments: <%= post.comments_counter %>,
+   Likes: <%= post.likes_counter %> </p>
+ </strong>
+ <ul>
+  <% post.comments.each do |comment| %>
+   <li> <%= comment.text %> </li>
+  <% end %>
+ </ul>
+</li>
```

Comment:
- [ ] You created this partial for the post index page. Kindly use the `recent_five_comments` method in your Post model, instead of rendering all comments.